test/public/index.html

```diff
@@ -5,6 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <script src="https://cdn.cardknox.com/sdk-3ds/1.0.2203.2501-beta/cardknox-3ds.min.js" integrity="sha256-SpY9bGtstDdMvc3jaeB/nW3sobN/NA8AqvigH1OoOow=" crossorigin="anonymous"></script>
```

Comment:
Is there a stable version we can use?

Reply:
Seems like we're only in beta
https://cdn.cardknox.com/sdk-3ds/versions.html